internal/command/local/connect.go

```diff
@@ -126,6 +119,13 @@ func runNonInteractiveConnect(log *slog.Logger, cfg *config.LocalConnect,
 
 	var cmd *exec.Cmd
 	if !cfg.Unprivileged {
+		if os.Geteuid() != 0 {
+			fmt.Printf("signadot local connect needs root priveleges for:\n\t" +
+				"- updating /etc/hosts with cluster service names\n\t" +
+				"- configuring networking to direct cluster traffic to the cluster\n" +
+				"If you would prefer to run without root priveleges, please type Control-C\n" +
+				"and run `signadot local connect --unprivileged`\n")
+		}
```

Comment:
This should be in the docs - let's not guess that the user wants to run without root in this case, because it seems more likely they'd end up here by accident.

Reply:
I have removed the reference to unprivileged use.  It now reads

```
23-07-12 scott@air cli % ./signadot local connect
signadot local connect needs root priveleges for:
	- updating /etc/hosts with cluster service names
	- configuring networking to direct cluster traffic to the cluster
Password:
```